modules/mono/csharp_script.cpp

```diff
@@ -3117,13 +3119,10 @@ MethodInfo CSharpScript::get_method_info(const StringName &p_method) const {
 }
 
 Error CSharpScript::reload(bool p_keep_state) {
-	bool has_instances;
-	{
-		MutexLock lock(CSharpLanguage::get_singleton()->script_instances_mutex);
-		has_instances = instances.size();
-	}
-
-	ERR_FAIL_COND_V(!p_keep_state && has_instances, ERR_ALREADY_IN_USE);
+	
+	if(!reload_invalidated) return OK;
```

Comment:
To follow the new code style guidelines, don't use single-line `if` statements, add braces and remove the blank line at the beginning:

```suggestion
	if (!reload_invalidated) {
		return OK;
	}
```